### DISCUSSION TITLE
Azure Table Storage

### DISCUSSION THREAD

**Discussion by TechAspirant** [serenity-is/Serenity#5457] [Help]
Hi,

Is their any connector or way I can show / Add / Delete data from azure table storage to serenity grid page ?

**Comment by VictorTomaili** [comment]
Currently we don't have a connector to azure table storage for community. You need implement your own with IUploadStorage interface.

**Comment by edwardch** [comment]
Isn't Azure Table Storage same/similar to Cosmos DB?

When you set up a new Cosmos DB you can pick the API type (SQL/Mongo/Cassandra/etc). As far as I know you should be able to treat the SQL API like a SQL connection, though I may be wrong about that...

**Reply by VictorTomaili** [reply]
Oh I see, my misunderstand. But also we don't have implementation of this.
